executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -178,8 +175,19 @@ pub fn split_out_machines<'a, T: FieldElement>(
             )));
         }
     }
+
+    // Always add a fixed lookup machine.
+    // Note that this machine comes last, because some machines do a fixed lookup
+    // in their take_witness_col_values() implementation.
+    // TODO: We should also split this up and have several instances instead.
```

Comment:
Would the different instances each have a consistent degree?

Reply:
Yes, would say so!